src/microbe_stage/editor/PatchMapDrawer.tscn

```diff
@@ -14,6 +14,14 @@ size_flags_horizontal = 3
 size_flags_vertical = 3
 script = ExtResource( 1 )
 DrawDefaultMapIfEmpty = true
-PatchNodeWidth = 60.0
-PatchNodeHeight = 60.0
+PatchNodeContainerPath = NodePath("PatchNodeContainer")
+LineContainerPath = NodePath("LineContainer")
 MonochromeMaterial = SubResource( 1 )
+
+[node name="LineContainer" type="Control" parent="."]
+margin_right = 40.0
```

Comment:
Should these have 0 marging to ensure they have the same draw area as the parent node?